tests/App/config.yml

```diff
@@ -44,13 +44,13 @@ sonata_media:
 
 fos_rest:
     param_fetcher_listener: true
-    body_listener: true
-    format_listener: true
-    view:
-        view_response_listener: force
-    body_converter:
-        enabled: false
-        validate: true
+#    body_listener: true
```

Comment:
Why are these lines commented?

Reply:
I resting drop sensio/framework-extra-bundle. Today is family day. I will finish it tomorrow.